README.md

```diff
@@ -40,11 +42,11 @@ To start using Sourcery on your code, check out our [Getting Started guide](http
 
 ![Refactoring Code with Sourcery](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/main/assets/Sourcery_VS_Code_Refactoring.gif)
 
-While you work, Sourcery will review all of the Python files you have open and look for opportunities to clean up and refactor your code. Once Sourcery finds a potential refactoring it will underline that section of your code.
+While you work, Sourcery will review all of the Python files you have open and look for opportunities to clean up and improve your code. Once Sourcery finds a potential improvement it will underline that section of your code.
 
-Hover your mouse over the underlined section of code to see the changes Sourcery suggests and see a diff of the proposed change.
+Hover your mouse over the underlined section of code to see the changes Sourcery suggests and to see a diff of the proposed change.
 
-To make the change, just bring up the quick fix menu and choose to accept the change. Sourcery will then apply the change to your code. We're constantly adding new refactorings that Sourcery can make. The current list of the types of refactorings Sourcery can make is available **[here](https://docs.sourcery.ai/refactorings/)**. 
+To make the change, just bring up the quick fix menu and choose to accept the change. Sourcery will then apply the change to your code. We're constantly adding new improvements and rules that Sourcery can make. The current list of rules Sourcery checks for is available **[here](https://docs.sourcery.ai/refactorings/)**.
```

Comment:
not quite sure this is the right link - it's to the refactorings (not the whole set of built ins)

Reply:
We don't have a link to the full set of built ins :/